### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by by-the-w3i** [by-the-w3i/MI-449-SS17-741-js-localstorage-wMUDce#1]
@KatieMFritz Can you take a look at this? It's [hosted here](https://jiangw14-kapstpq2uw6ovwmudce.surge.sh/) and meets the following criteria:

- [x] Includes a button that when clicked, will toggle the website between day and night themes
- [x] When a user changes the theme, their choice is remembered in `localStorage`
- [x] The class on the `body` element is always up-to-date with the current theme (either `day-theme` or `night-theme`) -- you can use `document.body.setAttribute('class', 'some-class-name')` to set the class on the `body`
- [x] When the body has the class `day-theme`, the background of the website is light-colored, with a darker color for the text
- [x] When the body has the class `night-theme`, the background of the website is a very dark color, with a lighter color for the text
- [x] Somewhere on the page, the user is told how many times they've visited the website (starting with `1` the first time)

**Comment by KatieMFritz** [comment]
Hi Wei, nice start with the day & night themes, but you're still missing some requirements. 🌞 🌑 

> When a user changes the theme, their choice is remembered in `localStorage`

When I refresh the page, it always goes back to day theme, even if I've changed it to night theme before refreshing.

>  Includes a button that when clicked, will toggle the website between day and night themes

After refreshing, I have to click "switch theme" twice to go from day theme to night theme.

> Somewhere on the page, the user is told how many times they've visited the website (starting with 1 the first time)

This updates every time I switch themes, but resets to 1 if I refresh the page. I would expect it to stay at the same number until I refresh, but then remember how many times I've visited before.

* * *
Let me know if you have questions about any requirements! Otherwise, make some changes, push them to GitHub and your hosted site, give it a once-over to make sure it looks right, then comment back here and I’ll take another look.

Thanks! :rocket:

**Comment by by-the-w3i** [comment]
Hi @KatieMFritz, I fixed the issues listed above.

**Comment by egillespie** [comment]
@by-the-w3i, this looks pretty good. Your `firstRun` is unnecessary, though. Browsers will reliably run scripts only one time so you don't need to explicitly track whether your code has already executed (and even if it did run again, your code would reset `firstRun` back to `true` on line 2). 😉 

Remove all the `firstRun` code and let me know when it's ready for me to take another look.

Thanks! 👻

**Comment by by-the-w3i** [comment]
@egillespie You are right, thank you for pointing this out. I just removed the unnecessary firstrun

**Comment by egillespie** [comment]
Awesome, thanks for the quick turn-around! :shipit:
